app/src/main/kotlin/com/exallium/todoapp/allnotes/AllNotesPresenter.kt

```diff
@@ -25,4 +30,10 @@ class AllNotesPresenter(private val view: AllNotesView,
         adapter.cleanup()
     }
 
-}
\ No newline at end of file
+    private fun makeNoteDetailBundle(id: String?) : Bundle {
+        val args: Bundle = Bundle()
```

Comment:
I think I like the idea of utilizing a `BundleFactory` here to prevent calling the constructor inside presenter code.  That way we can decouple ourselves and dont need to bring in Robolectric to test it.